content/en/docs/10.0/quotas.md

```diff
@@ -217,7 +217,7 @@ Near the end of the output you can find the relevant status part:
 ```
   containerStatuses:
   - containerID: docker://da2473f1c8ccdffbb824d03689e9fe738ed689853e9c2643c37f206d10f93a73
-    image: polinux/stress:latest
+    image: docker-registry.mobicorp.ch/polinux/stress:latest
```

Comment:
I just realize this should either be put into another switch where we differentiate the image output or we simply use `polinux/stress:latest` as generic example.